base/logging.jl

```diff
@@ -671,7 +671,7 @@ function handle_message(logger::SimpleLogger, level::LogLevel, message, _module,
         remaining > 0 || return
     end
     buf = IOBuffer()
-    stream = logger.stream
+    stream::IO = logger.stream
```

Comment:
`logger.stream` is declared as `IO`-field -- so best to annotate L676 as `stream = stderr::IO`, but it's no longer necessary for v1.9.